micropub.php

```diff
@@ -187,7 +188,7 @@ private static function map_params($q) {
       $args['post_date'] = iso8601_to_datetime($q['published']);
     }
 
-    return $args;
+    return apply_filter('before_micropub', $args, $q);
```

Comment:
mind moving this into `parse_query`, after the `map_params` call? it's subjective, but i'd prefer it there.